ha/core/config/config_manager.py

```diff
@@ -72,16 +72,16 @@ def _conf_init(config_file):
         ConfigManager._safe_load(const.HA_GLOBAL_INDEX, config_file)
 
     @staticmethod
-    def get_confstore():
+    def get_confstore(kv_enable_batch = False):
         """
-        Initalize and get confstore if not _cluster_confstore is None.
-        Used by config manager methods to check and initalize confstore if needed.
+        Initialize and get confstore if _cluster_confstore is None.
+        Used by config manager methods to check and initialize confstore if needed.
         """
         if ConfigManager._cluster_confstore is None:
             consul_endpoint = Conf.get(const.HA_GLOBAL_INDEX, f'consul_config{_DELIM}endpoint')
             consul_host = consul_endpoint.split(":")[1].strip("//")
             consul_port = consul_endpoint.split(":")[-1]
-            ConfigManager._cluster_confstore = ConsulKvStore(prefix=const.CLUSTER_CONFSTORE_PREFIX, host=consul_host, port=consul_port)
+            ConfigManager._cluster_confstore = ConsulKvStore(prefix=const.CLUSTER_CONFSTORE_PREFIX, host=consul_host, port=consul_port, enable_batch=kv_enable_batch)
```

Comment:
Wow, this is a super long statement, this cannot be printed on a dot matrix ;).
Kindly try to maintain a 80 column limit please.

Reply:
This seems to be through out the code though.

Reply:
yes, I will try to maintain the 80 limit.
but still, do we need to have an 80 limit as sometimes it looks ugly and now we have extended/long monitors.
it's my thought.

Reply:
changed